intersection_manager/main.py

```diff
@@ -158,16 +159,20 @@ def main(args):
         traffic_lights[i].set_state(carla.TrafficLightState.Red)
 
     # initiate logging
-    zenoh.init_logger()
-
-    logging.info('Opening session...')
-    session = zenoh.open(conf)
-
-    logging.info("Declaring Queryable on '{}'...".format(key))
-    _queryable = session.declare_queryable(key, queryable_callback, complete)
-
-    while True:
-        time.sleep(1)
+    zenoh.try_init_log_from_env()
+
+    print("Opening session...")
```

Comment:
I prefer to use the consistent way to do logging.
Which one do you think is better, `print` or `logging.info`?